deploy/csi-operator.yaml

```diff
@@ -926,7 +946,7 @@ spec:
             - name: OPENEBS_NAMESPACE
               value: openebs
             - name: REMOUNT
-              value: "false"
+              value: "true"
```

Comment:
Lets not make it as default right now. There is no test case around this.

Reply:
done